services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -496,13 +502,9 @@ async def get_project(
         only_published: bool = False,
         only_templates: bool = False,
     ) -> tuple[ProjectDict, ProjectType]:
-        """Returns all projects *owned* by the user
-
-            - prj_owner
-            - Notice that a user can have access to a template but he might not own it
-            - Notice that a user can have access to a project where he/she has read access
-
-        :raises ProjectNotFoundError: project is not assigned to user
```

Comment:
could you please bring this back

Reply:
Those comments were already not valid. User id and access were removed from this function in some previous PR.